z/allocator.go

```diff
@@ -56,8 +56,8 @@ func init() {
 	allocs = make(map[uint64]*Allocator)
 
 	// Set up a unique Ref per process.
-	rand.Seed(time.Now().UnixNano())
-	allocRef = uint64(rand.Int63n(1<<16)) << 48 //nolint:gosec // cryptographic precision not needed
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
We are ignoring the Rand source returned by `rand.New`. In this case, we can simply remove this line and continue to use the global source. I created a PR here https://github.com/dgraph-io/ristretto/pull/385